sre-operational-fabric.md

```diff
@@ -0,0 +1,71 @@
+
+Our team supports this mission by improving
+[observability](https://en.wikipedia.org/wiki/Observability) (i.e. insight
+into) and [controllability](https://en.wikipedia.org/wiki/Controllability)
+(i.e. state mutation) of cloud deployments located in datacenters throughout
+the world.
+
+## Role
+
+Site Reliability Engineering (SRE) is an engineering discipline that combines
+software and systems engineering to build and run large-scale, massively
+distributed, fault-tolerant systems. SRE ensures that Rackspace's managed
+service offerings & customer deployments have reliability and uptime
+appropriate to users' needs and a fast rate of improvement while monitoring and
+validating capacity and performance. Act as a Technical Lead for a portfolio of
+managed service deployments. Focused on reliability, scalability and the
+development of automation to manage a set of repetitive tasks at scale.
+
+## About Us
+
+- **Team Size**: Currently 8 senior developers/engineers and a technical manager
+- **Hours**: We have flexible working hours and aim for a median of 35-40 hours a week
+- **On-Call**: No, we have a 24x7 ops team and escalations (uncommon) go to management
+- **Location**: US/UK work from home with offices in US/UK (most states supported)
+- **Travel**: About once or twice a year for a conference or team gathering
+
+## Responsibilities
+
+- Help develop and maintain *observability*/controllability of customer environments
+- Help drive zero downtime (0DD) CI/CD for our teams' infrastructure and app stack
+- Work directly with support engineers, operations, product management, and end-users
+- Develop automation to improve efficiency, reliability, and simplicity of clouds
+- Take high level problems and autonomously deliver solutions supported by data
+- Develop solutions that leverage and integrate existing open source technologies
+- Engage asynchronously with team members via Slack, Jira, github, and email
+- Engage synchronously with team members via video calls
+
+## Qualifications
+
+The ideal candidate will have the following (**not** hard requirements):
+
+- Experience defining architecture of production systems that run at scale
+- Experience developing and deploying applications at scale
+- Experience working on distributed teams
+- Experience with 0DD CI/CD on Kubernetes using Helm
+- Experience with multiple cloud computing technologies (e.g. AWS, GCP, OpenStack)
+- Experience with multiple programming paradigms (i.e. declarative, functional, imperative, object-oriented)
+- A polyglot (e.g. Python, Go, Haskell)
+- Good working knowledge of network security and secure coding fundamentals (e.g. SSL, GIGO)
+- Good working knowledge of messaging technologies (e.g. RabbitMQ, Kafka)
+- Good working knowledge of document, relational, and time-series databases (e.g. Mongo, MySQL, Influx)
+- Good working knowledge of monitoring technologies (e.g. Prometheus, TICK)
+- Excellent written and verbal skills
```

Comment:
Maybe something about experience working to automate away problems that cause RCAs, if they happen. I'm not sure how to phrase it, but basically the ability to handle production issues as they occur while mainly focusing on automating prevention and mitigation of failures.

Depending on how involved our team will be in some aspects of the stack, maybe operating multiple production environments simultaneously will be good. It's easier to work on 1 homogeneous environment compared to the task we're undertaking.

Reply:
I think that's generically caught in the automation bullet. Also, added all the standard SRE bullets so probably covered.

Reply:
Let me know if you still think it needs added with updates made

Reply:
s/production systems/multiple production systems for emphasis in the first bullet here and I'm +1. I think that's worth the emphasis since it's a pretty unique situation IMHO.